lib/plugins/aws/configCredentials/awsConfigCredentials.js

```diff
@@ -64,19 +65,13 @@ class AwsConfigCredentials {
     this.serverless.cli.log('Setting up AWS...');
     this.serverless.cli.log('Saving your AWS profile in "~/.aws/credentials"...');
 
-    // locate home directory on user's machine
-    const env = process.env;
-    const home = env.HOME ||
-      env.USERPROFILE ||
-      (env.HOMEPATH ? ((env.HOMEDRIVE || 'C:/') + env.HOMEPATH) : null);
-
-    if (!home) {
+    if (!os.homedir()) {
```

Comment:
Here's the list of Node versions we "officially" support: https://github.com/serverless/serverless/blob/53c96088a2f15c72173536ab94428c8ea5f4b7d2/.travis.yml#L5-L8